src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IArgument.vb

```diff
@@ -1074,5 +1074,269 @@ End Class]]>.Value
 
             VerifyClone(model)
         End Sub
+
+        <Fact>
```

Comment:
If I remember correctly, VB not only has named, un-named and omitted arguments, it also has "range" arguments. I'm not sure the syntax.

Reply:
added.